contracts/clients/package.json

```diff
@@ -16,9 +16,12 @@
     "publish-experimental": "node scripts/showVersion.js >.version && npm version $(node scripts/showBaseVersion.js)-$(git rev-parse HEAD | head -c7) --allow-same-version && npm publish --tag experimental && npm version $(cat .version) && rm .version",
     "publish-experimental-dry-run": "node scripts/showVersion.js >.version && npm version $(node scripts/showBaseVersion.js)-$(git rev-parse HEAD | head -c7) --allow-same-version && npm publish --tag experimental --dry-run && npm version $(cat .version) && rm .version"
   },
+  "resolutions": {
+    "mcl-wasm": "1.0.0"
+  },
```

Comment:
@jacque006 Remove with new `@thehubbleproject/bls` release.

Reply:
Will do this in a follow up PR or if a new version of `@thehubbleproject/bls` is released while this is in flight.